lib/gomod-parser.ts

```diff
@@ -173,3 +177,61 @@ export function toSnykVersion(v: ModuleVersion): string {
     return (v as ModuleExactVersion).exactVersion;
   }
 }
+
+export async function buildDepTreeFromImports(path: string = '.') {
+  const depTree: DepTree = {
+    name: 'tempName', // TODO: How do get name of the module?
+    version: 'tempVersion', // TODO: How do get version of the module?
+    dependencies: {},
+  };
+  const goDepsOutput = await spawnGoListCommand(path);
+  if (goDepsOutput.includes('matched no packages')) {
+    return depTree;
+  }
+  const objectBorderRegexp = new RegExp(`}${EOL}{`, 'g');
+  const goDepsString = `[${goDepsOutput.replace(objectBorderRegexp, '},{')}]`;
+  const goDeps: [GoPackage] = JSON.parse(goDepsString);
```

Comment:
umm shouldn't the type be `GoPackage[]` ?

Reply:
It should, 🤦‍♂ on my side.